packages/commonwealth/server/routes/updateBanner.ts

```diff
@@ -9,11 +9,12 @@ enum UpdateBannerErrors {
   NoPermission = `You do not have permission to update banner`,
 }
 
-type UpdateBannerReq = Omit<CommunityBannerInstance, 'id'> & {
+type UpdateBannerReq = {
   community_id: string;
   banner_text: string;
 };
 
+// XXX: this route should be rolled into the updateCommunity route
```

Comment:
can you create a ticket and add the number here instead? otherwise this comment is not actionable